web/packages/teleport/src/features.tsx

```diff
@@ -67,6 +67,23 @@ import { NavTitle, type FeatureFlags, type TeleportFeature } from './types';
 import { UnifiedResources } from './UnifiedResources';
 import { Users } from './Users';
 
+// to promote feature discoverability, most features should be visible in the navigation even if a user doesnt have access.
+// However, there are some cases where hiding the feature is explicitly requested. Use this as a backdoor to hide the features that
+// are usually "always visible"
+export function shouldHideFromNavigation(cfg: {
+  isTeam: boolean;
+  isUsageBasedBilling: boolean;
```

Comment:
Most subscriptions nowadays should have `isUsageBasedBilling=true` IIRC. The main exceptions are OSS and 14-day trials.

This means that `shouldHideFromNavigation` will return `true` for most customers, is this what we want?